ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1065,9 +1250,24 @@ export const RuntimePanel = fp.flow(
               </FlexRow>
             </Fragment>
       ],
-      [PanelContent.Delete, () => <ConfirmDelete
+      [PanelContent.Delete, () => (enablePD && selectedCompute === ComputeType.Standard && !runtimeExists && pdExists) ?
```

Comment:
I'm thinking whether DELETE_PD should be its own PanelContent entry. I think it probably should - it is possible we would need to route differently depending on what we're doing, and we can't always assume delete = delete runtime or PD